velox/duckdb/conversion/DuckParser.h

```diff
@@ -86,6 +86,11 @@ struct IExprWindowFunction {
       orderBy;
 };
 
-const IExprWindowFunction parseWindowExpr(const std::string& windowString);
+// Parse a WINDOW function SQL string. We pass parse options to this
```

Comment:
Nit: There isn't any need for this comment here. The user can pass any ParseOptions as needed, so the comment only documents a particular use of this API.